test-cases/cruftdirectory/cruft-directory-test-cases.sh

```diff
@@ -3,9 +3,9 @@ loopmax=0
 . ../common.sh
 
 [[ $silentMode == 0 ]] && set -x 
-latexindent.pl -s  cruft-directory1 -c ~/Desktop/tmp/ -w -g=one.log
-latexindent.pl -s  cruft-directory1 -c ~/Desktop/tmp/ -w -g=two.log 
-latexindent.pl -s  cruft-directory1 -c ~/Desktop/tmp/ -w -l=more-than-one-back-up.yaml -g=three.log 
+latexindent.pl -s  cruft-directory1 -c $(mktemp -d) -w -g=one.log
```

Comment:
The default container has no user besides root; consequently, `~/Desktop` does not exist.